public/javascripts/app/controllers/SnapshotContentCtrl.js

```diff
@@ -97,6 +100,19 @@ SnapshotContentCtrlMod.controller('SnapshotContentCtrl', [
       mediator.publish('snapshot-list:display-modal');
     }
 
+    this.copyJSON = function() {
+      const sillyHacks = document.createElement("textarea");
```

Comment:
What is this world we live in.

Reply:
I tried the whizzy new Clipboard API but it doesn't work in Firefox :(

https://www.w3.org/TR/clipboard-apis/#async-clipboard-api
https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript